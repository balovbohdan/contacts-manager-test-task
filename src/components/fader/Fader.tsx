import * as React from 'react';
import {Fade} from 'react-reveal';

export const Fader = ({children}) =>
    <Fade timeout={400}>{children}</Fade>;