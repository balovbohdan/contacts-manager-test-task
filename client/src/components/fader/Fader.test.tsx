import * as React from 'react';
import {shallow} from 'enzyme';

import {Fader} from './Fader';

describe('Fader', () => {
    it('Should match snapshot', () => {
        const fader = shallow(<Fader>Foo</Fader>);

        expect(fader).toMatchSnapshot();
    });
});