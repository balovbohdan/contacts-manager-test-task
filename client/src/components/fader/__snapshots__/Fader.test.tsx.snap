// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fader Should match snapshot 1`] = `
<Fade
  timeout={400}
>
  Foo
</Fade>
`;
